spec/duration.html

```diff
@@ -1508,6 +1512,7 @@ <h1>
           _calendar_: an Object,
           _relativeTo_: a Temporal.PlainDate,
           _duration_: a Temporal.Duration,
+          _dateAdd_: a function object or *undefined*,
```

Comment:
```suggestion
          optional _dateAdd_: a function object,
```
I think this is preferred for optional parameters to abstract operations?

Reply:
The last commit changes `dateAdd` to a required parameter, because all callers pass `dateAdd`. We have to allow `undefined`, because `GetMethod` returns either a function or `undefined`.

Reply:
Got it, thanks.